config/v2/config.go

```diff
@@ -146,9 +146,13 @@ type SnowflakeProvider struct {
 	Version *string `json:"version,omitempty" yaml:"version,omitempty"`
 }
 
-type HerokuProvider struct{}
+type HerokuProvider struct {
+	Version *string `yaml:"version,omitempty"`
```

Comment:
I wonder if we should have a BaseProvider or something like it that we embed into these providers

Reply:
👍